src/components/GordonDialogBox/index.js

```diff
@@ -38,9 +38,16 @@ const GordonDialogBox = ({
   children,
   ...otherProps
 }) => {
+  const handleClose = (event, reason) => {
+    if (typeof cancelButtonClicked === 'function') {
+      cancelButtonClicked();
+    }
+  };
```

Comment:
This is the recommended way to resolve the deprecation of backdrop-related Dialog hooks. See https://robertmarshall.dev/blog/disablebackdropclick-of-forwardrefdialog-is-deprecated-solution-and-example/

Reply:
I don't think this accomplishes anything besides preventing `cancelButtonClicked` from being called if it's not a function (which is not an awful thing, but that would only apply when someone passes something that isn't a function and is not `undefined` as the value of `cancelButtonClicked`).

Are you just trying to get the dialog to close when the backdrop is clicked? If so, the simplest solution is to pass `onClose` to the `GordonDialogBox`, rather than changing the default behavior of the `GordonDialogBox`.

Reply:
Yea, that's essentially what it was doing except also checking the type. I will remove that type check.